evm/src/witness/transition.rs

```diff
@@ -184,8 +184,8 @@ fn fill_op_flag<F: Field>(op: Operation, row: &mut CpuColumnsView<F>) {
         Operation::Jumpi => &mut flags.jumpi,
         Operation::Pc => &mut flags.pc,
         Operation::Jumpdest => &mut flags.jumpdest,
-        Operation::GetContext => &mut flags.get_context,
-        Operation::SetContext => &mut flags.set_context,
+        Operation::GetContext => &mut flags.context_op,
+        Operation::SetContext => &mut flags.context_op,
```

Comment:
Nit: For consistency, I'd like to make sure that each flag column appears on the right only once.
```suggestion
        Operation::GetContext | Operation::SetContext => &mut flags.context_op,
```